e2e/testcases/acme_test.go

```diff
@@ -45,15 +44,18 @@ func configSyncManagementLabels(namespace, folder string) map[string]string {
 func TestAcmeCorpRepo(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.Reconciliation1)
 
+	rootSyncGitRepo := nt.SyncSourceGitRepository(nomostest.DefaultRootSyncID)
```

Comment:
nit: If we wanted to break this into two smaller PRs we could potentially break out all of the `nt.SyncSourceGitRepository` refactors to a separate change (the first change just being a wrapper for `nt.RootRepos`/`nt.NamespaceRepos`). Might make this a bit easier to review.

Reply:
Separating those out across a few hundred tests might take a couple hours... because the old code doesn't use IDs or SyncSource. So let me know if you think it's actually gonna save you a few hours reviewing.

Reply:
I was thinking it could use the same ID argument to return a Repository, but just use RootRepos/NamespaceRepos under the hood - that way the follow up just needs to replace the `SyncSourceGitRepository` implementation to use the new SyncSources. I'm almost done reviewing though so no biggie